test/scripts/dump_credentials.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Dumps the contents of the environment variable GOOGLE_CREDENTIALS_BASE64 to
+ * a file.
+ *
+ * To setup Travis to run on your fork, read TRAVIS.md.
+ */
+$cred = getenv('GOOGLE_CREDENTIALS_BASE64');
```

Comment:
I think I remember this... At the time it was because there was an earlier check for it.